src/main/java/org/dimamir999/dao/FileDao.java

```diff
@@ -10,36 +12,47 @@
 import java.util.List;
 
 public class FileDao {
+    private static final Logger log = LogManager.getLogger(FileDao.class);
 
-    public String read(String fileName) throws IOException{
+    public String read(String fileName) throws IOException {
         StringBuilder data = new StringBuilder();
+
         try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
             String currentLine = null;
-            while ((currentLine = br.readLine()) != null) {
+            while((currentLine = br.readLine()) != null) {
                 data.append(currentLine + "\n");
             }
         }
+
         return data.toString();
     }
 
     public boolean write(String data, String fileName) throws IOException {
         BufferedWriter bw = null;
         FileWriter fw = null;
+
         try {
             fw = new FileWriter(fileName);
             bw = new BufferedWriter(fw);
             bw.write(data);
+        } catch (IOException var14) {
```

Comment:
Rename exception var14 to ex or e. And add here more convenient message smth like "file can`t be read"